src/lib/optimizer/strategy/between_composition_rule.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
```

Comment:
The comments in this file can be improved. I have no idea what a column boundary is, why it needs to hold onto a predicate node and why it has a ValueExpression.

Reply:
Please check again if the documentation is sufficient now